### ISSUE TITLE
php-pdo-oci - Errno::ENAMETOOLONG: File name too long @ rb_sysopen

### ISSUE THREAD

**Issue by nohnaimer** [kabel/homebrew-php-ext#9]
Hi,
After update php 7.2.11 to 7.2.12 not updated php-pdo-oci. Any ideas how fix?
```
$ php -v
PHP 7.2.12 (cli) (built: Nov  9 2018 11:00:45) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.2.0, Copyright (c) 1998-2018 Zend Technologies
    with Xdebug v2.6.1, Copyright (c) 2002-2018, by Derick Rethans
    with Zend OPcache v7.2.12, Copyright (c) 1999-2018, by Zend Technologies

$ brew install php-pdo-oci
==> Installing php-pdo-oci from kabel/php-ext
==> Downloading https://php.net/get/php-7.2.12.tar.xz/from/this/mirror
Already downloaded: /Users/and1/Library/Caches/Homebrew/downloads/55b491e15e68a5aa001cebb8ad0cbb934dfdf962e14b567d8de5eab1af400871--php-7.2.12.tar.xz
Error: An exception occurred within a child process:
  Errno::ENAMETOOLONG: File name too long @ rb_sysopen - /usr/local/var/homebrew/locks/665aa2952dd4fcdbbe25f6a02ee3cc8cf5b39ab36c8001447b303fe567cc8354--Site2pstoreToken=v1.2~EC062A1473FF0629A4~22C469B3A88830BF297972CEC4D040D4D7B255DF615439CBEB6E50C4AC0BFC3DC412D1BD04233B1CDEA842C4DA9CC3C4E2A0026D9B778C4BF3B46D6A70FDBA3CEB0970C37239D86DECA3437FA4448D57AA8CD0FABC09AF2E6175A92256A414B70A706269D7EF2A6BAB132F4A7248D03C599A05DA86B771607133DB5E3648B72344DCF9E582CCD6633794A58E38A4227E10D385B60CFFB8585182816F9F2E93EF0C5B6D18F5A06C0BECD5108996A57C70EFC0DC0CEF5994BCAF5479180E06C97C10EC20989AA4308515EF71646A9931235BA2142C28FD5D450242211127001212543488BAB65AEA21BE46086B72F7EE45A98883C0E6B1FE07498E85271BF975A67863D54C461E7005EE6D3C098F1ED79B2353169D59E3D78B.incomplete.lock
```

**Comment by kabel** [comment]
I'll clear my local cache and take a look later. If I had to venture a quick guess, there were some changes to the way brew is naming temporary formula resources which now makes the file name too long.

The workaround would be to manually download the oracle instantclient and instantclient-sdk and put them into the Homebrew cache in the location brew expects.

**Comment by kabel** [comment]
Fixed at 53cbe4ebc34382d6ef0a88fc58f0f9e875888a50
